README.md

```diff
@@ -1,25 +1,37 @@
-# Javascript code prettifier
+# JavaScript code prettifier
 
 An embeddable script that makes source-code snippets in HTML prettier.
 
-[See an example](https://rawgit.com/google/code-prettify/master/examples/quine.html)
+* Works on HTML pages.
+* Works even if code contains embedded links, line numbers, etc.
+* Simple API: include some JS & CSS and add an onload handler.
+* Lightweights: small download and does not block page from loading while
+  running.
+* Customizable styles via CSS. See the [themes gallery][1].
+* Supports all C-like, Bash-like, and XML-like languages. No need to specify
+  the language.
+* Extensible language handlers for other languages. You can specify the
+  language.
+* Widely used with good cross-browser support. Powers https://code.google.com/
+  and http://stackoverflow.com/
+
+[See an example][2].
```

Comment:
I actually lifted those from the old wiki pages :)
https://github.com/google/code-prettify/blob/wiki/ProjectHome.md